config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -200,7 +202,7 @@ periodics:
     preset-service-account: "true"
     preset-common-gce-windows: "true"
     preset-e2e-gce-windows: "true"
-    preset-windows-repo-list-2004: "true"
+    preset-windows-repo-list-master: "true"
```

Comment:
For 2004 job, I think we want to use image-repo-list-2004 instead because the test image in the default gcAuthenticatedRegistry does not support 2004, so we have to manually point it to e2eprivate dockerhub.

Reply:
do we have an open PR to get the latest versions into gcAuthenticated?

Reply:
@claudiubelu  do you have any update on this? Ideally, we would only want to maintain one image-repo-list for master branch instead of two.